### PR TITLE
Add Amplify deployment to NextJs example

### PR DESCRIPTION
**Pull request by guiyom-e**
Starting from `feat/swarmion-with-next` branch, I added an option to deploy the NextJs 13 app to Amplify, with a CDK construct.
See the updated README for more details on setup and use.

Commit to be reviewed (@guillaumeduboc) :
- chore: add amplify to deploy nextjs app

Thanks to @alexandrepernin who first deployed a POC of this new deployment option!

image

### CODE REVIEW

**Comment by guillaumeduboc** [comment]
I am changing the base of this PR to `feat/swarmion-with-next`

File: examples/swarmion-with-next/frontend/next-app/package.json [hunk 1]
```diff
@@ -3,14 +3,17 @@
   "private": true,
   "version": "1.0.0",
   "scripts": {
+    "bootstrap": "cdk bootstrap",
```
Comment (guillaumeduboc) [inline examples/swarmion-with-next/frontend/next-app/package.json:6]: can you ensure that nx runs bootstrap before deploy?
there is an example [here](https://github.com/swarmion/swarmion/pull/351)

File: examples/swarmion-with-next/frontend/next-app/README.md [hunk 2]
```diff
@@ -27,8 +27,51 @@ To learn more about Next.js, take a look at the following resources:
...
+- Whenever a branch starts with `feature/`, a preview will be built and deployed.
+  It is possible to update this branch preview configuration in [`branchSettings.ts`](./hosting/settings/branchSettings.ts)
+
+To include the app deployment in an advanced Continuous Deployment pipeline, you may use [webhooks](https://docs.aws.amazon.com/amplify/latest/userguide/webhooks.html). There is no CDK configuration available for now, but some commands are available in the [AWS CLI](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/amplify/create-webhook.html) to automate the process.
+
 ## Deploy on Vercel
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+Another way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
```
Comment (fargito) [inline examples/swarmion-with-next/frontend/next-app/README.md:75]: Could you remove the UTM parameters from the link?

**Review by adriencaccia — CHANGES_REQUESTED** [review]
Can you change the commit message to something like `examples(with-next): ...` so that it appears nicely in the release notes?
Thanks 🙏

File: examples/swarmion-with-next/frontend/next-app/package.json (hunk 1 above)
Comment (guiyom-e) [reply examples/swarmion-with-next/frontend/next-app/package.json:6]: Yes, I added the nx configuration to cache and run the command!

**Comment by fargito** [comment]
Waiting for #347

**Pull request closed without merging**